src/backend/joanie/core/models/certifications.py

```diff
@@ -153,15 +159,33 @@ def _set_localized_context(self):
         title_object = (
             self.order.product if self.order else self.enrollment.course_run.course
         )
-
+        new_images = set()
         for language, __ in settings.LANGUAGES:
             context[language] = {
                 "course": {
                     "name": title_object.safe_translation_getter(
                         "title", language_code=language
                     ),
                 },
-                "organizations": [
+                "organizations": [],
+            }
+
+            for organization in self.course.organizations.all():
+                signature_checksum = file_checksum(organization.signature)
+                (signature, _) = DocumentImage.objects.get_or_create(
+                    checksum=signature_checksum,
+                    defaults={"file": organization.signature},
+                )
+                new_images.add(signature)
+
+                logo_checksum = file_checksum(organization.logo)
```

Comment:
You don't check before if `organization.logo` exists ?

Reply:
Yup. It's missing.